### PR TITLE
#5 Developer should be able to see JS and CSS lint errors when testing

### PR DESCRIPTION
**Pull request by yanarchy**
Add scaffolding and lint and contribution info to README. Add eslint and stylelint set up files. Change and add test commands to package.json.

### CODE REVIEW

**Review by billfeng — APPROVED** [review]
LGTM!

**Review by saba96 — APPROVED** [review]

**Comment by kumilange** [comment]
@yanarchy 
link to airMap needs to be corrected

**Comment by kumilange** [comment]
@bill-feng 
Maybe from next time, we need a review approval  from 2 people + YOU before merge.
> At least two people on the team AND the product manager must review and accept the pull request before merging.

**Pull request merged**
